### DISCUSSION TITLE
Dose any one know a bunch of unblocked websites?

### DISCUSSION THREAD

**Discussion by winghf** [3kh0/ext-remover#231] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Give me your websites bellow :D

**Comment by ILikeBypassing** [comment]
[this repo](https://github.com/Bigfoot9999/Bigfoots-Game-Shack) has a bunch of unblocked games

**Reply by 3kh0** [reply]
Yeah but this one is better tbh: https://github.com/3kh0/3kh0.github.io

**Reply by ILikeBypassing** [reply]
well my school blocks all urls ending with ".io" so i cant use that
maybe you can make a google sites?

**Reply by 3kh0** [reply]
deploy a copy of the site

**Reply by ILikeBypassing** [reply]
oh yeah the source code is public... good idea

**Reply by Aka-but-nice** [reply]
> oh yeah the source code is public... good idea

What extension do you have?

**Comment by Sxri** [comment]
I'm working on https://anchorx.lol its not done yet or no where near done but its ok

**Reply by KyceAbdallah** [reply]
dam its blocked 💀

**Comment by ghost** [comment]
> Give me your websites bellow :D

Try out my Gsite https://www.norvid.ml

**Reply by KyceAbdallah** [reply]
it gives me a 404 error
![image](https://user-images.githubusercontent.com/90739617/228569471-13ecd2c5-888b-4bd5-88be-36dbfe09cba0.png)

**Comment by TrickJoker312** [comment]
My GitHub repository has over 100 unblocked games and proxies for you to enjoy! Here's the link: https://github.com/TrickJoker312/Unblocked-Sites

**Reply by KyceAbdallah** [reply]
Any chance you can slightly change the name of the repository? Cuz my admin blocked anything with "unblocked" in the link so it says blocked. Even if it slightly misspells "unblocked" is fine

**Reply by 3kh0** [reply]
@TrickJoker312 your thing does not even work

**Reply by TrickJoker312** [reply]
@3kh0 May you please explain how it doesn't work? I'd like to help support the community by fixing bugs, so everyone can enjoy unblocked games!
PS. If one site is blocked, it doesn't mean every one is.

**Comment by 3kh0** [comment, marked as answer]
Try this, https://3kh0.github.io

**Reply by KyceAbdallah** [reply]
Nope it blocked it for me 💀

**Reply by PSYclownYT** [reply]
ur website is blocked lol. I forked and was copy-striked

**Reply by viperbot2000** [reply]
https://3kh0-mirrors.github.io/
it has all of da links

**Reply by viperbot2000** [reply]
or this https://3kh0-rahib777-7.vercel.app/
